rmgpy/data/kinetics/database.py

```diff
@@ -408,7 +408,7 @@ def generate_reactions_from_library(self, library, reactants, products=None):
 
         return reaction_list
 
-    def generate_reactions_from_families(self, reactants, products=None, only_families=None, resonance=True):
+    def generate_reactions_from_families(self, reactants, products=None, only_families=None, resonance=True, clear_labels=True):
```

Comment:
I'm actually planning on removing the commits related to the clear_labels argument, since it didn't work as expected. In the convert library to training ipynb, we generate reactions from families then regenerate the atom labels, so I thought it would be easier to not clear the labels to begin with so we wouldn't need to regenerate them. It turned out to be much more complicated though.